docs/self-managed/concepts/multi-region/dual-region.md

```diff
@@ -0,0 +1,198 @@
+
+In an **active-active** setup, multiple application instances run simultaneously in different regions, actively handling user requests. This allows for better load balancing and fault tolerance, as traffic can spread across regions. If one region fails, the workload can shift to another without causing disruptions.
+
+In contrast, an **active-passive** setup designates one region as the main or active region where all user requests are processed. The other region remains on standby until needed, only becoming active if the previously active region fails. This setup is easier to manage but may result in higher delays during failover events.
+
+### User traffic
+
+The overall system is **active-passive**, even though some components may be **active-active**. You will have to take care of the user traffic routing or DNS by yourself, and won't be considered further. You must select one region as the actively serving region and route the user traffic there. In case of a total region failure, you must route the traffic to the passive region yourself.
+
+<!-- Should we provide some reading materials on how to tackle this? -->
+
+### Components
+
+#### Zeebe
+
+Zeebe, as a workflow engine, is fully **active-active** and replicates data between its brokers. Due to its replication logic, it can handle an entire region failure without data loss. Requiring a proper partition and replication setup. Consider reading through the [cluster technical concept](./../../../components/zeebe/technical-concepts/clustering.md) to learn more about the [Raft protocol](<https://en.wikipedia.org/wiki/Raft_(algorithm)>).
+
+#### Elasticsearch
+
+We treat Elasticsearch as an **active-passive** component. While it may be possible to deploy it in active-active mode and stretch across regions, it's not officially documented by Elasticsearch. Such a setup generally brings its own challenges along that we can't control and may negatively impact the performance of Zeebe.
+
+We go with the approach of having one Elasticsearch per region and configuring Zeebe to export the Elasticsearch data in both regions.
+
+#### WebApps
+
+WebApps are an **active-passive** component and in its current state also not highly available since we're limited by the included exporters that would cause data issues when running multiple instances at the same time.
+
+In every region, there can be only a single WebApp instance running (i.e. in Region 0, 1 instance of Operate, 1 instance of Tasklist; same in Region 1)
+
+This means that one instance will be actively serving traffic, while the other one is on standby.
+
+##### Operate
+
+In the event of a total active region loss, the following data will be lost:
+
+- Batch Operations <!-- Do we have a docs link for this? -->
+
+##### Tasklist
+
+In the event of a total active region loss, the following data will be lost:
+
+- Assignments of tasks
```

Comment:
should this be "assigments of uncompleted tasks"? As afaik the completed ones are preserved in Zeebe.